acme/http_challenge_webserver.go

```diff
@@ -0,0 +1,85 @@
+package acme
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"strings"
+)
+
+type HttpChallengeWebserver struct {
+	optPort string
+	start   chan net.Listener
+	end     chan error
+}
+
+func NewHttpChallengeWebserver(optPort string) HttpChallengeMethod {
```

Comment:
For some reason, the word "Webserver" is really bothering me - maybe because it's actually two words. Can we change this to `NewHTTPChallengeWebServer` instead with a capital S and a capital HTTP?